### ISSUE TITLE
amp-analytics click triggers do not work in FIE.

### ISSUE THREAD

**Issue by avimehta** [ampproject/amphtml#7212]
A config like: 

```
  {
    "requests": {"click-px-0":"https://example.com/dyn?clickRequest"},
    "triggers": {
      "click-pixels": {
        "on": "click",
        "selector": "#clk",
        "request": "click-px-0"
      }
    },
    "transport": {
      "beacon": false,
      "xhrpost": false,
      "image": true
    }
  }
  ```

triggers when in a normal cross domain iframe but not in A4A FIE.

/cc @dvoytenko @lannka if you guys can get to this first.

**Comment by jasti** [comment]
@keithwrightbos why is this not an issue with AMP creatives we generate?
@szach FYI

**Comment by dvoytenko** [comment]
This should be fixed by recent #7172. Could you please confirm?
